homeassistant/components/modbus/const.py

```diff
@@ -64,12 +64,20 @@
 CONF_HVAC_MODE_REGISTER = "hvac_mode_register"
 CONF_HVAC_ONOFF_REGISTER = "hvac_onoff_register"
 CONF_HVAC_MODE_OFF = "state_off"
+CONF_HVAC_MODE_OFF_TT_REG = "state_off_target_temp_register"
 CONF_HVAC_MODE_HEAT = "state_heat"
+CONF_HVAC_MODE_HEAT_TT_REG = "state_heat_target_temp_register"
 CONF_HVAC_MODE_COOL = "state_cool"
+CONF_HVAC_MODE_COOL_TT_REG = "state_cool_target_temp_register"
 CONF_HVAC_MODE_HEAT_COOL = "state_heat_cool"
+CONF_HVAC_MODE_HEAT_COOL_TT_REG = "state_heat_cool_target_temp_register"
 CONF_HVAC_MODE_AUTO = "state_auto"
+CONF_HVAC_MODE_AUTO_TT_REG = "state_auto_target_temp_register"
 CONF_HVAC_MODE_DRY = "state_dry"
+CONF_HVAC_MODE_DRY_TT_REG = "state_dry_target_temp_register"
 CONF_HVAC_MODE_FAN_ONLY = "state_fan_only"
+CONF_HVAC_MODE_FAN_ONLY_TT_REG = "state_fan_only_target_temp_register"
+CONF_HVAC_MODE_TT_REG_BY_VALUES = "hvac_mode_target_temp_registers"
```

Comment:
Const indicates its a value, but text indicates its a list of registers...one of those is wrong.

Reply:
@janiversen no problem, I'll change also the const name and already done for the text.